src/buttons.hpp

```diff
@@ -6,10 +6,40 @@
 
 #include "commands.hpp"
 #include "constants.hpp"
+/*
+three Button
+============
+                                                      vol/next-prev inverted
+           admin-menu  idle        pause       play        play
+-------------------------------------------------------------------------
+pause      select                  play        pause       pause
+pauseLong  end adm     shortcut    shortcut    track#      track#
+up         next                                next        vol++
+upLong     next 10     shortcut    shortcut    vol++       next
```

Comment:
Actually, upLong with inverted buttons is implemented as next, *cont.* now. Is that what you wanted?

Reply:
I think now it is implemented only as next, without repetition